torch/_refs/__init__.py

```diff
@@ -951,7 +951,10 @@ def flatten(a: TensorLikeType, start_dim: int = 0, end_dim: int = -1) -> TensorL
 
 @register_decomposition(torch.ops.aten.flip, register_meta=True)
 def flip(a: TensorLikeType, dims: DimsSequenceType) -> TensorLikeType:
+    if not isinstance(dims, tuple) and not isinstance(dims, list):
```

Comment:
nit: we should create a DimsSequence so we can `isinstance(dims, DimsSequence)` analogous to other custom types